src/Utils/FindUserByUserId.ts

```diff
@@ -0,0 +1,6 @@
+import { User } from '../Types/user';
+import usersFromServer from './../api/users';
```

Comment:
[CODE KNOWLEDGE] - The `import` keyword is used in a non-root component to import `usersFromServer`. According to the checklist, you should prepare data in one place (App component) and pass it to child components instead of using `import` in non-root components.